scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl

```diff
@@ -633,3 +651,115 @@ task SchemaNullTranscriptsExist {
+
+    command <<<
+        set -e
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+            gsutil cp ~{service_account_json_path} local.service_account.json
+            gcloud auth activate-service-account --key-file=local.service_account.json
+            gcloud config set project ~{query_project_id}
+        fi
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
+
+        # gvs subpopulations:  [ "afr", "amr", "eas", "eur", "mid", "oth", "sas"]
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT
+            vid
+        FROM
+            ~{fq_vat_table}
+        WHERE
+            gvs_max_af < gvs_afr_af OR
+            gvs_max_af < gvs_amr_af OR
+            gvs_max_af < gvs_eas_af OR
+            gvs_max_af < gvs_eur_af OR
+            gvs_max_af < gvs_mid_af OR
+            gvs_max_af < gvs_oth_af OR
+            gvs_max_af < gvs_sas_af'
+
+        # if the result of the query has any rows, that means gvs_max_af is not in fact the max af
+        if [[ $NUMRESULTS != "0" ]]; then
+          echo "PASS: The VAT table ~{fq_vat_table} has a correct calculation for subpopulation" > validation_results.txt
+        else
+          echo "FAIL: The VAT table ~{fq_vat_table} has an incorrect calculation for subpopulation" > validation_results.txt
+        fi
+    >>>
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:305.0.0"
+        memory: "1 GB"
+        preemptible: 3
+        cpu: "1"
+        disks: "local-disk 100 HDD"
+    }
+    # ------------------------------------------------
+    # Output: {"Name of validation rule": "PASS/FAIL plus additional validation results"}
+    output {
+        Map[String, String] result = {"SubpopulationMax": read_string('validation_results.txt')}
+    }
+}
+
+task SubpopulationAlleleCount {
+    input {
+        String query_project_id
+        String fq_vat_table
+        String? service_account_json_path
+        String last_modified_timestamp
+    }
+    # sum of subpop ACs equal the gvs_all ACs
+    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+
+    command <<<
+        set -e
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+            gsutil cp ~{service_account_json_path} local.service_account.json
+            gcloud auth activate-service-account --key-file=local.service_account.json
+            gcloud config set project ~{query_project_id}
+        fi
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
+
+        # gvs subpopulations:  [ "afr", "amr", "eas", "eur", "mid", "oth", "sas"]
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT
+            vid
+        FROM
+            ~{fq_vat_table}
+        WHERE
+            gvs_all_ac != 20
```

Comment:
This will all get ripped out during de-obfuscation pr -- it was really just so that there wouldn't be validation run at all on the rows where the AC had been obfuscated to 20 (because the math wouldn't check out there)